includes/discovery/ports.inc.php

```diff
@@ -64,7 +82,7 @@
             $ports_db[$port_id]['deleted'] = '0';
             echo 'U';
         } else { // port is existing, let's update it with some data we have collected here
-            dbUpdate(array('ifType' => $ifType, 'ifName' => $ifName, 'ifAlias' => $ifAlias, 'ifDescr' => $ifDescr), 'ports', '`port_id` = ?', array($port_id));
```

Comment:
Why did you remove ifAlias update?

Reply:
If you set the description manualy in the webinterface the ifAlias is changed in the DB. If the discover is running after this it is updating it with the info from the device. And so the manual set description (ifAlias in DB) is reverted back and the manual changed setting is gone.

Reply:
Interesting.  Is it worth checking the device attribute to add this info on first discover and not clobber it later?

Reply:
Yep, that would make sense in that case. I was not aware of the manual setting as I do not use it but, the set of ifalias should end up in a dedicated function that does this check anytime an update is tried.

Reply:
```php
        if (!empty($device['attribs']['ifName:' . $port['ifName']])) {
            // ifAlias overridden by user, don't update it
            unset($snmp_data['ifAlias']);
        }
```

(this assumes you've extracted the arrays to a var called $snmp_data)